.mailmap

```diff
@@ -1593,6 +1593,7 @@ gregmedlock <gmedlo@gmail.com>
 helo9 <helo9@users.noreply.github.com>
 hm <hacman0@gmail.com>
 immerrr <immerrr@gmail.com>
+jamestitus299 <titusjames299@gmail.com> jamestitus299 <jamestitus299@gmail.com>
```

Comment:
Take a look at the [AUTHORS](https://github.com/sympy/sympy/blob/master/AUTHORS) file.

Do you want your name to be recorded there as "jamestitus299" or as something like "James Titus"?